src/pytorch_lightning/strategies/hivemind.py

```diff
@@ -315,6 +315,7 @@ def __init__(self, optimizer: "hivemind.Optimizer", scheduler: _LRScheduler) ->
         # implemented custom logic which we would not want to call on destruction of the `HiveMindScheduler`
         self.__dict__ = {k: v for k, v in scheduler.__dict__.items() if k not in ("step", "__del__")}
 
+        self.base_lrs = scheduler.base_lrs
```

Comment:
This should be already set from the `__dict__` override 2 lines above

Reply:
That is true, but I think mypy does not pick it up since it's set by `__dict__` override

Reply:
But then we probably should just add stubs before the init instead of doing that again

Reply:
I did it. Thank you for the advice!